pkg/provider-local/imagevector/images.go

```diff
@@ -7,10 +7,10 @@
 package imagevector
```

Comment:
Any reason why this file was not renamed to `containers.go` analogous to the corresponding file in `imagevector` directory?

Reply:
See https://github.com/gardener/gardener/pull/10218#discussion_r1695058323